src/odemis/driver/orsay.py

```diff
@@ -2149,7 +2149,7 @@ def __init__(self, name, role, parent, **kwargs):
         + mirrorImage: BooleanVA, True to mirror the retrieved image
         + imageFromSteerers: BooleanVA, True to image from Steerers, False to image from Octopoles
         + objectiveVoltage: FloatContinuous, unit="V", range=(0.0, 2e4)
-        + beamShift: TupleContinuous Float, unit=m, range=[(-1.0e-4, -1.0e-4), (1.0e-4, 1.0e-4)]
+        + shift: TupleContinuous Float, unit=m, range=[(-1.0e-4, -1.0e-4), (1.0e-4, 1.0e-4)]
```

Comment:
very minor comment: just to keep it consistent with the row above and below, could you change the unit to "m"?